Mathlib/Data/Multiset/Powerset.lean

```diff
@@ -140,6 +137,7 @@ theorem revzip_powersetAux' {l : List α} ⦃x⦄ (h : x ∈ revzip (powersetAux
   exact Quot.sound (revzip_sublists' _ _ _ h)
 #align multiset.revzip_powerset_aux' Multiset.revzip_powersetAux'
 
+universe u in
```

Comment:
It should be possible to change `Type u` below to `Type*` and delete this line. (And delete the porting note.)